extras/common/app/views/kaminari/blacklight/_next_page.html.erb

```diff
@@ -0,0 +1,26 @@
+<%# Link to the "Next" page
```

Comment:
Added this file in order to put `disabled` attribute on pagination Next link (CSS class .disabled will not suffice).